skins/moono-lexicon/icons/icons.json

```diff
@@ -4,36 +4,59 @@
         "align-image-center": "align-image-center",
         "align-image-left": "align-image-left",
         "align-image-right": "align-image-right",
-        "anchor": "flag-full",
+        "anchor": {
+            "ltr": "flag-full"
```

Comment:
Same here... why just `ltr`? 

I just have no idea what this is supposed to do, it just feels weird :D

Reply:
Same as the others